python/meterpreter/tests/test_ext_server_stdapi.py

```diff
@@ -320,6 +320,27 @@ def test_stdapi_sys_config_getsid(self):
         ).get("value")
         self.assertRegex(sid, "S-1-5-.*")
 
+class ExtStdNetResolveTest(ExtServerStdApiTest):
+    def test_stdapi_net_resolve_host(self):
```

Comment:
Probably worth adding a resolve_hosts since we're here and it'll be a pretty easy add as it's going to use the same building blocks as this current function